### PR TITLE
[Polkadot] estimate bond unbond rebond

### PR DESCRIPTION
**Pull request by henri-ly**

### CODE REVIEW

File: src/families/polkadot/js-estimateMaxSpendable.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  parentAccount: ?Account,
+  transaction: ?Transaction,
+}): Promise<BigNumber> => {
+  const mainAccount = getMainAccount(account, parentAccount);
+  const t = {
+    ...createTransaction(),
+    ...transaction,
+    recipient:
+      transaction?.recipient ||
+      "1Z4QdzRrpVbggYoGK5pfbeMyzpVVDK7WxheVjWFxfv6sxjV", // need abandon seed and being empty
```
Comment (haammar-ledger) [inline src/families/polkadot/js-estimateMaxSpendable.js:28]: Shouldn't we remove this now?

Comment (henri-ly) [reply src/families/polkadot/js-estimateMaxSpendable.js:28]: Not really it's a fallback when we don't have recipient, the command estimateMaxSpendable doesn't include a transaction.

Comment (haammar-ledger) [reply src/families/polkadot/js-estimateMaxSpendable.js:28]: got it :+1:

**Review by haammar-ledger — APPROVED** [review]
LGTM

**Pull request merged**